src/urllib3/connectionpool.py

```diff
@@ -64,29 +72,31 @@ class ConnectionPool:
        target URIs.
     """
 
-    scheme = None
+    scheme: Optional[str] = None
     QueueCls = queue.LifoQueue
 
-    def __init__(self, host, port=None):
-        if not host:
+    def __init__(self, host: Optional[str], port: Optional[int] = None) -> None:
+        if host is None:
```

Comment:
This changes functionality for empty hosts, we should keep this branch the same.

Reply:
changed reverted.